package.json

```
@@ -95,5 +95,8 @@
     "ts-jest": "28.0.2",
     "ts-loader": "9.3.0",
     "ts-node": "10.7.0"
+  },
+  "lint-staged": {
+    "packages/**/*.{ts,json}": []
```

lol I just notice that I copy-paste this from `nestjs/nest` but doesn't make sense here as we don't have the `packages/` dir

```suggestion
    "**/*.{ts,json}": []
```

---

True, i thought about it too i usually use this:

```
  "lint-stage": {
    "* .ts": [
      "prettier - write"
    ]
  },
```
in the two packages for nestjs I manage.

but i think removing "packages" should be fine

---

I think husky should signal this with a warning that the path is wrong, too bad he doesn't.